gearcatalog/frontend/src/components/accounts/__tests__/Login.test.tsx

```diff
@@ -9,7 +9,7 @@ jest.mock('react-router-dom', () => ({
   useHistory: () => ({
     push: mockHistoryPush,
   }),
-  Link: 'Link',
+  Link: 'div',
```

Comment:
What is this change for?

Reply:
When I was running the tests it was giving me an `console.error` stating that react elements need to be pascal case. So I just changed the mock to a built in element.